.ci_support/linux_.yaml

```diff
@@ -25,16 +25,14 @@ mpi:
 mpi4py:
 - 3.0.1
 numpy:
-- 1.16.*
+- '1.14'
```

Comment:
I don't think this is the way to go forward. I think what we really want here is to pin it using:
- {{ pin_compatible('numpy') }}
See: https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy